app/javascript/components/modals/area-of-interest/component.jsx

```diff
@@ -29,7 +30,9 @@ class AreaOfInterestModal extends PureComponent {
   };
 
   render() {
-    const { open, loading, loggedIn, canDelete, viewAfterSave } = this.props;
+    const { open, loading, userData, canDelete, viewAfterSave } = this.props;
+    const { email, fullName, lastName, loggedIn } = userData || {};
+    const isProfileFormFilled = !!email && (!!fullName || !!lastName);
```

Comment:
you don't need to do this; `email && (fullName || lastName)` will work just fine